src/security-council-mgmt/SecurityCouncilNominationsGovernor.sol

```diff
@@ -0,0 +1,225 @@
+        _;
+    }
+
+    ////////////// GovernorCountingSimple Overrides //////////////
+
+    function COUNTING_MODE() public pure virtual override(GovernorCountingSimpleUpgradeable, IGovernorUpgradeable) returns (string memory) {
+        return "TODO: ???";
+    }
+
+    function hasVoted(
+        uint256 proposalId, 
+        address account
+    ) 
+        public 
+        view 
+        override(GovernorCountingSimpleUpgradeable, IGovernorUpgradeable) 
+        returns (bool) 
+    {
+        // should this return true if they've cast any amount of votes? or if they've cast all of their votes?
+        revert("TODO");
+    }
+
+    // unsupported because it is supposed to return for, against, abstain. but that doesn't make sense for nominations
+    function proposalVotes(
+        uint256
+    ) public pure override(GovernorCountingSimpleUpgradeable) returns (uint256, uint256, uint256) {
+        revert("Unsupported");
+    }
+
+    // there is no minimum quorum for nominations, so we just return true
+    function _quorumReached(uint256) internal pure override(GovernorCountingSimpleUpgradeable, GovernorUpgradeable) returns (bool) {
+        return true;
+    }
+
+    // the vote always succeeds, so we just return true
+    function _voteSucceeded(uint256) internal pure override(GovernorCountingSimpleUpgradeable, GovernorUpgradeable) returns (bool) {
+        return true;
+    }
+
+    function _countVote(
+        uint256 proposalId,
+        address account,
+        uint8,
+        uint256 weight,
+        bytes memory params
+    ) internal virtual override(GovernorCountingSimpleUpgradeable, GovernorUpgradeable) {
+        // let's say params is (address candidate, uint256 tokens)
+        (address candidate, uint256 tokens) = abi.decode(params, (address, uint256));
+
+        require(_isCandidateEligible(proposalId, candidate), "Candidate is not eligible");
+
+        NominationElection storage election = nominationElections[proposalId];
+
+        // weight is the number of tokens that account has at the time of the vote
+        // make sure tokens + previously used tokens is less than or equal to weight
+        uint256 previouslyUsedTokens = election.nominationsTokensUsed[account];
+        require(tokens + previouslyUsedTokens <= weight, "Not enough tokens to cast this vote");
+
+        // add to nominationsTokensUsed
+        election.nominationsTokensUsed[account] = previouslyUsedTokens + tokens;
+
+        // add tokens to the candidate
+        uint256 oldVotesForCandidate = election.nominationsVotes[candidate];
+        election.nominationsVotes[candidate] = oldVotesForCandidate + tokens;
+
+        // if this vote put the candidate over the line, push to successfullyNominatedCandidates
+        uint256 votesNeeded = quorum(proposalSnapshot(proposalId));
+        if (oldVotesForCandidate < votesNeeded && oldVotesForCandidate + tokens >= votesNeeded) {
+            election.successfullyNominatedCandidates.push(candidate);
```

Comment:
Maybe we can semantically disambiguate the different types of "nominees"; i.e., one option is use the Oscar's terms: "contenders" (submitted but < .2%), "nominees" (>.2% of the votes), and then "members / council members"   (winners of the phase 3 election)

Reply:
I like that a lot, Oscar's terms sound good!